docs/source/chaincode4ade.rst

```diff
@@ -18,58 +18,35 @@ only read query is allowed. That is, the called chaincode on a different channel
 which does not participate in state validation checks in subsequent commit phase.
 
 In the following sections, we will explore chaincode through the eyes of an
-application developer. We'll present a simple chaincode sample application
-and walk through the purpose of each method in the Chaincode Shim API. If you
+application developer. We'll present a asset-transfer chaincode sample walkthrough,
+and the purpose of each method in the Fabric Contract API. If you
 are a network operator who is deploying a chaincode to running network,
 visit the :doc:`deploy_chaincode` tutorial and the :doc:`chaincode_lifecycle`
 concept topic.
 
-This tutorial provides an overview of the low level APIs provided by the Fabric
-Chaincode Shim API. You can also use the higher level APIs provided by the
-Fabric Contract API. To learn more about developing smart contracts
-using the Fabric contract API, visit the :doc:`developapps/smartcontract` topic.
+This tutorial provides an overview of the high level APIs provided by the Fabric Contract API.
+To learn more about developing smart contracts using the Fabric contract API, visit the :doc:`developapps/smartcontract` topic.
 
-Chaincode API
--------------
+Fabric Contract API
+-------------------
 
-Every chaincode program must implement the ``Chaincode`` interface whose methods
-are called in response to received transactions. You can find the reference
-documentation of the Chaincode Shim API for different languages below:
+The ``fabric-contract-api`` provides the contract interface, a high level API for application developers to implement Smart Contracts.
+Within Hyperledger Fabric, Smart Contracts are also known as Chaincode. Working with this API provides a high level entry point to writing business logic.
```

Comment:
`Smart Contracts are also known as Chaincode.`

This is not really true anymore. We now have different definitions for the two items. Chaincode is the package that is deployed, while the code is smart contracts. We should make that distinction here, maybe to the point of renaming the tutorial.